frontend/src/metabase/dashboard/hooks/use-dashboard-url-params.ts

```diff
@@ -40,11 +40,6 @@ export const useDashboardUrlParams = ({
     theme,
   } = useEmbedTheme();
 
-  const normalizedTheme = normalizeTheme({
```

Comment:
moved the logic to `frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboardView.tsx`, so I could test the logic with loki